web/src/components/routes/rallyists/ArtistsList.tsx

```
@@ -1,5 +1,6 @@
+import { useNavigate } from "@tanstack/react-router";
 import clsx from "clsx";
-import { TicketCheck } from "lucide-react";
+import { Map, TicketCheck } from "lucide-react";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)